src/java/org/apache/cassandra/config/RepairRetrySpec.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.config;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class RepairRetrySpec extends RetrySpec
+{
+    public RetrySpec.Partial merkle_tree_response = null;
+
+    public boolean isMerkleTreeResponseEnabled()
```

Comment:
this should be renamed - makes it sound like we can disable merkle tree responses

also the other comment in this file is not addressed yet